### ISSUE TITLE
WCAG 2.2 Violation - There are instances of incorrect/missing focus movement (after user-initiated actions) present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#14]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [2.4.3 Focus Order](https://www.w3.org/TR/WCAG22/#focus-order)

#### Further information:

1. 

### Instances

1. #52
